packages/shared/src/utils/contract/contract.tsx

```diff
@@ -29,15 +29,67 @@ export const DELAY_TIME_1S_SYMBOL = 500;
 // generation_interval will be provided via API later to help us distinguish between 1-second and 2-second symbols
 export const symbols_2s = ['R_10', 'R_25', 'R_50', 'R_75', 'R_100'];
 
-export const TURBOS = {
-    LONG: 'turboslong',
-    SHORT: 'turbosshort',
+export const CONTRACT_TYPES = {
+    ACCUMULATOR: 'ACCU',
+    ASIAN: { UP: 'ASIANU', DOWN: 'ASIAND' },
+    CALL_PUT_SPREAD: { CALL: 'CALLSPREAD', PUT: 'PUTSPREAD' },
+    END: { IN: 'EXPIRYRANGE', OUT: 'EXPIRYMISS' },
+    EVEN_ODD: { ODD: 'DIGITODD', EVEN: 'DIGITEVEN' },
+    HIGH_LOW: { CALL: 'CALL', PUT: 'PUT' },
+    LB_HIGH_LOW: 'LBHIGHLOW',
+    LB_CALL: 'LBFLOATCALL',
+    LB_PUT: 'LBFLOATPUT',
+    MATCH_DIFF: { MATCH: 'DIGITMATCH', DIFF: 'DIGITDIFF' },
+    MULTIPLIER: {
+        UP: 'MULTUP',
+        DOWN: 'MULTDOWN',
+    },
+    OVER_UNDER: { OVER: 'DIGITOVER', UNDER: 'DIGITUNDER' },
+    RESET: { CALL: 'RESETCALL', PUT: 'RESETPUT' },
+    RISE_FALL: { CALL: 'CALL', PUT: 'PUT' },
+    RISE_FALL_EQUAL: { CALLE: 'CALLE', PUTE: 'PUTE' },
+    RUN_HIGH_LOW: { HIGH: 'RUNHIGH', LOW: 'RUNLOW' },
+    STAY: { IN: 'RANGE', OUT: 'UPORDOWN' },
+    TICK_HIGH_LOW: { HIGH: 'TICKHIGH', LOW: 'TICKLOW' },
+    TOUCH: { ONE_TOUCH: 'ONETOUCH', NO_TOUCH: 'NOTOUCH' },
+    TURBOS: {
+        LONG: 'TURBOSLONG',
+        SHORT: 'TURBOSSHORT',
+    },
+    VANILLA: {
+        CALL: 'VANILLALONGCALL',
+        PUT: 'VANILLALONGPUT',
+    },
 } as const;
-
-export const VANILLALONG = {
-    CALL: 'vanillalongcall',
-    PUT: 'vanillalongput',
-    FX: 'vanilla_fx',
+export const TRADE_TYPES = {
+    ACCUMULATOR: 'accumulator',
+    ASIAN: 'asian',
+    CALL_PUT_SPREAD: 'callputspread',
+    END: 'end',
+    EVEN_ODD: 'even_odd',
+    HIGH_LOW: 'high_low',
+    LB_HIGH_LOW: 'lb_high_low',
+    LB_CALL: 'lb_call',
+    LB_PUT: 'lb_put',
+    MATCH_DIFF: 'match_diff',
+    MULTIPLIER: 'multiplier',
+    OVER_UNDER: 'over_under',
+    RESET: 'reset',
+    RISE_FALL: 'rise_fall',
+    RISE_FALL_EQUAL: 'rise_fall_equal',
+    RUN_HIGH_LOW: 'run_high_low',
+    STAY: 'stay',
+    TICK_HIGH_LOW: 'tick_high_low',
+    TOUCH: 'touch',
+    TURBOS: {
+        LONG: 'turboslong',
```

Comment:
```suggestion
        LONG: CONTRACT_TYPES.TURBOS.LONG.toLowerCase(),
```
@kate-deriv  it's ok to reuse them ;)